conduit/pipeline.go

```diff
@@ -507,6 +502,15 @@ func MakePipeline(ctx context.Context, cfg *PipelineConfig, logger *log.Logger)
 	if logger == nil {
 		return nil, fmt.Errorf("MakePipeline(): logger was empty")
 	}
+
+	if cfg.LogFile != "" {
+		f, err := os.OpenFile(cfg.LogFile, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0644)
+		if err != nil {
+			return nil, fmt.Errorf("MakePipeline(): %w", err)
+		}
+		logger.SetOutput(f)
+	}
```

Comment:
We can probably add a really simple unit test that just ensures the expected log file gets created in the location we tell it to here.

Reply:
added a test.